signingscript/docker.d/passwords.yml

```diff
@@ -277,24 +270,6 @@ in:
 
       # passwords-mobile.json
       'ENV == "prod" && COT_PRODUCT == "mobile"':
-        # TODO: Remove the following once fenix is in the monorepo
-        project:mobile:fenix:releng:signing:cert:production-signing:
-          - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_FENIX_USERNAME"},
-             {"$eval": "AUTOGRAPH_FENIX_PASSWORD"},
-             ["autograph_apk"]
-            ]
-          - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_FENIX_MOZILLA_ONLINE_USERNAME"},
-             {"$eval": "AUTOGRAPH_FENIX_MOZILLA_ONLINE_PASSWORD"},
-             ["autograph_apk_mozillaonline"]
-            ]
-        project:mobile:fenix:releng:signing:cert:fennec-production-signing:
-          - ["https://autograph-external.prod.autograph.services.mozaws.net",
-             {"$eval": "AUTOGRAPH_FENNEC_RELEASE_USERNAME"},
-             {"$eval": "AUTOGRAPH_FENNEC_RELEASE_PASSWORD"},
-             ["autograph_apk"]
-            ]
         project:mobile:fennec-profile-manager:releng:signing:cert:fennec-nightly-signing:
```

Comment:
Unrelated to this PR. We don't need this scope anymore. The fennec-profile-manager was a test app we used when we migrated Fennec users to Fenix. I'll file a follow-up bug about it. 

[1] https://github.com/mozilla-mobile/fennec-profile-manager